code/modules/paperwork/desk_bell.dm

```diff
@@ -0,0 +1,91 @@
+// A receptionist's bhell
+
+/obj/structure/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	layer = OBJ_LAYER
+	anchored = FALSE
+	pass_flags = PASSTABLE // Able to place on tables
+	max_integrity = 5000 // To make attacking it not instantly break it
+	/// The amount of times this bell has been rang, used to check the chance it breaks
+	var/times_rang = 0
+	/// Is this bell broken?
+	var/broken_ringer = FALSE
+	/// The cooldown for ringing the bell
+	COOLDOWN_DECLARE(ring_cooldown)
+	/// The length of the cooldown. Setting it to 0 will skip all cooldowns alltogether.
+	var/ring_cooldown_length = 0.3 SECONDS // This is here to protect against tinnitus.
+	/// The sound the bell makes
+	var/ring_sound = 'sound/machines/microwave/microwave-end.ogg'
+
+/obj/structure/desk_bell/Initialize(mapload)
+	. = ..()
+
+
+/obj/structure/desk_bell/attack_hand(mob/living/user, list/modifiers)
+	. = ..()
+	if(!COOLDOWN_FINISHED(src, ring_cooldown) && ring_cooldown_length)
+		return TRUE
+	if(!ring_bell(user))
+		to_chat(user,"<span class='notice'>[src] is silent. Some idiot broke it.</span>")
+	if(ring_cooldown_length)
+		COOLDOWN_START(src, ring_cooldown, ring_cooldown_length)
+	return TRUE
+
+/obj/structure/desk_bell/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	times_rang += I.force
```

Comment:
Stuff that does 0 damage should still increment it, but won't they just cause integrity damage directly and break it anyway?

Reply:
I think it's intended, as it might be consider as "tampering" with it